src/Resources/contao/dca/tl_calendar_events.php

```diff
@@ -670,7 +670,18 @@ public function generateAlias($varValue, DataContainer $dc)
 		if ($varValue == '')
 		{
 			$autoAlias = true;
-			$varValue = StringUtil::generateAlias($dc->activeRecord->title);
+			$slugOptions = [];
+			$objCalendar = CalendarModel::findByPk($dc->activeRecord->pid);
```

Comment:
We should handle the case `$objCalendar === null`, too.

Reply:
I think you are right :)